### PR TITLE
React Implementation - Sprint 1

### PR DESCRIPTION
**Pull request by OliOliKT**

### CODE REVIEW

File: src/components/CommentComponent.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const CommentBox = () => {
   return (
     <div>
       <textarea
-        id="comments"
+        id="comment"
```
Comment (mircealungu) [inline src/components/CommentComponent.js:33]: Element id should be unique in the app. This looks very _un-unique_.

File: src/components/PostFilter.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import './PostFilter.css';
```
Comment (mircealungu) [inline src/components/PostFilter.js]: Nice that you're importing the styles from the corresponding css file!

File: src/components/PostFilter.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+function PostFilter() {
+    return (
+        <div id="filter">
```
Comment (mircealungu) [inline src/components/PostFilter.js]: id should be unique.

File: src/pages/IndividualPost.css [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.IndividualPost {
```
Comment (mircealungu) [inline src/pages/IndividualPost.css]: https://github.com/itu-tid/lecture-notes/blob/main/Tech-TopUps/5-Coding-Practices.md#name-your-css-classes-with-kebab-case

**Review by mircealungu — CHANGES_REQUESTED** [review]
overall good. a few requests for change

**Comment by emhberg** [comment]
Some of these changes, like variable names and kebab case will be changes for the next sprint.

File: src/components/PostFilter.js (hunk 3 above)
Comment (jdisitu) [reply src/components/PostFilter.js]: on it

**Pull request closed without merging**